Sources/Sentry/include/SentryFileContents.h

```diff
@@ -7,7 +7,7 @@ NS_ASSUME_NONNULL_BEGIN
 - (instancetype)initWithPath:(NSString *)path andContents:(NSData *)contents;
 
 @property (nonatomic, readonly, copy) NSString *path;
-@property (nonatomic, readonly, weak) NSData *contents;
+@property (nonatomic, readonly, strong) NSData *contents;
```

Comment:
`m`: Can we write a test to reproduce the issue we had to avoid having this problem in the future?

Reply:
Tbh I don't know how to properly test this right now.
I will put this in our backlog to not forget.